### PR TITLE
Docs: document that the reporting task needs access to NiFi to run

### LINKED ISSUE #562: Documentation: the NiFi operator creates a Job that connects to the NiFi stacklet
This needs to be documented.

### PR DESCRIPTION
**Pull request by fhennig**
# Description

fixes #562 

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes

```[tasklist]
# Reviewer
- [x] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/style-guide).
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

### CODE REVIEW

File: docs/modules/nifi/pages/usage_guide/monitoring.adoc [hunk 1]
```diff
@@ -1,4 +1,8 @@
 = Monitoring
 
-The managed NiFi cluster is automatically configured to export Prometheus metrics. See
-xref:home:operators:monitoring.adoc[] for more details.
+The operator automatically configures NiFi to export Prometheus metrics.
+This is done by creating a https://kubernetes.io/docs/concepts/workloads/controllers/job/[Job] that connects to NiFi and configures a reporting task.
```
Comment (siegfriedweber) [inline docs/modules/nifi/pages/usage_guide/monitoring.adoc:4]: On some documentation pages, an external link icon is used to mark an external link:

```suggestion
This is done by creating a https://kubernetes.io/docs/concepts/workloads/controllers/job/[Job^{external-link-icon}^] that connects to NiFi and configures a reporting task.
```

I am not sure if this should be applied to all external links.

File: docs/modules/nifi/pages/usage_guide/monitoring.adoc [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+
+IMPORTANT: network access from the Job to NiFi is required. If you are running a Kubernetes with restrictive NetworkPolicies, make sure to allow access from the Job to NiFi.
```
Comment (siegfriedweber) [inline docs/modules/nifi/pages/usage_guide/monitoring.adoc:6]:
```suggestion
IMPORTANT: Network access from the Job to NiFi is required. If you are running a Kubernetes with restrictive NetworkPolicies, make sure to allow access from the Job to NiFi.
```

**nit:** Is a link to NetworkPolicies useful?

**Review by siegfriedweber — CHANGES_REQUESTED** [review]

File: docs/modules/nifi/pages/usage_guide/monitoring.adoc (hunk 1 above)
Comment (fhennig) [reply docs/modules/nifi/pages/usage_guide/monitoring.adoc:4]: I've experimented with this icon in some parts, yes. but it was mostly for menu entries. I would not use it here now, but I think it's probably best to come up with a policy about it and put it in the style guide. I'm making a note of it

**Review by siegfriedweber — APPROVED** [review]
LGTM

**Pull request merged**